src/components/DashboardNoTeam/InitialPage/InitalPage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Text, Button, Center, Heading, VStack } from '@chakra-ui/react';
+
+const InitialPage = props => {
```

Comment:
I think it'd be useful to destructure all props so it's easy for other developers to see exactly what props are passed in
```suggestion
const InitialPage = props => {
    const { x, y, z } = props;
```